src/PluginBase/RadiumPluginInterface.hpp

```diff
@@ -106,6 +113,17 @@ namespace Ra
              * @return The created and configured feature widget
              */
             virtual QWidget* getFeatureTrackingWidget() { return nullptr; }
+
+            virtual bool doAddRenderer() { return false; }
+
+            /**
+             * @brief addRenderers
+             *
+             * \param rds the TODO
+             * \warning Allocated renderers are given to the application and
+             * SHOULD not be destroyed by the plugin
+             */
+            virtual void addRenderers(std::vector<Engine::Renderer*> */*rds*/) {}
```

Comment:
TODO : Use shared_ptr to share the ownership of renderers between plugin and viewer